content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -84,7 +84,7 @@ is the [rewrite-target annotation](https://github.com/kubernetes/ingress-nginx/b
 Different [Ingress controllers](/docs/concepts/services-networking/ingress-controllers) support different annotations.
 Review the documentation for your choice of Ingress controller to learn which annotations are supported.
 
-The Ingress [spec](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status)
+The [Ingress spec](/docs/reference/kubernetes-api/service-resources/ingress-v1/#IngressSpec)
```

Comment:
The original link for spec is the api convention of spec not specific to Ingress.

I would link to https://kubernetes.io/docs/concepts/overview/working-with-objects/#object-spec-and-status

Reply:
> The Ingress spec has all the information needed to configure a load balancer or proxy server. Most importantly, it contains a list of rules matched against all incoming requests. Ingress resource only supports rules for directing HTTP(S) traffic.

@sftim but The content in the paragraph is regarding ingress spec not related api convention of spec, so it will be good if it will redirect specifically to the Ingress.

Reply:
The intent here is that the reader understands the jargon term “spec”.

Even better: use a glossary tooltip for spec. But: you first need to add “spec” into our [glossary](https://kubernetes.io/docs/reference/glossary/?all=true).

Reply:
OK, @sftim then I feel updating the link to https://kubernetes.io/docs/concepts/overview/working-with-objects/#object-spec-and-status seems good.

Reply:
regarding your comment on the [corresponding issue 43105](https://github.com/kubernetes/website/issues/43105#issuecomment-1727490611)

> I don't agree with the proposed solution.
> 
> Try to link to concept pages for each thing we want to explain
> If feasible, drop the links to https://kubernetes.github.io/ingress-nginx/* from the concept page.

there aren't any concept pages that contain information for them, it is related to Ingress Nginx controller ({kubernetes/ingress-nginx](https://github.com/kubernetes/ingress-nginx)). So I linked it to the corresponding docs of it.

Any other way, if it seems not good.

Reply:
@sftim your thoughts?

Reply:
I would still have linked to https://kubernetes.io/docs/concepts/overview/working-with-objects/#object-spec-and-status. As for dropping links, that's less important.